.github/workflows/python-package.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          npm install pyright
```

Comment:
Is this line necessary when we have it listed in the hook's `additional_dependencies` section?

Reply:
Maybe not. I can try without.

Reply:
Works without the line, so have removed it.